services/incentive/src/main/java/org/cryptimeleon/incentive/services/incentive/error/OnlineDoubleSpendingException.java

```diff
@@ -0,0 +1,7 @@
+package org.cryptimeleon.incentive.services.incentive.error;
+
+public class OnlineDoubleSpendingException extends IncentiveServiceException {
+    public OnlineDoubleSpendingException() {
+        super("");
```

Comment:
same question here

Reply:
See above :)
But other error code such that the app can tell what happened